assets/com.github.th-ch.youtube-music.metainfo.xml

```diff
@@ -0,0 +1,423 @@
+<?xml version='1.0' encoding='utf-8'?>
+<component type="desktop-application">
+  <id>com.github.th_ch.youtube_music</id>
+  <name>YouTube Music</name>
+  <summary>YouTube Music Desktop App - including custom plugins</summary>
+  <developer_name>th-ch</developer_name>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>MIT</project_license>
+  <description>
+    <p>Electron wrapper around YouTube Music featuring:</p>
+    <ul>
+      <li>Native look &amp; feel, aims at keeping the original interface</li>
+      <li>Framework for custom plugins: change YouTube Music to your needs (style, content,
+        features), enable/disable plugins in one click</li>
+    </ul>
+  </description>
+  <screenshots>
+    <screenshot type="default">
+      <caption>Preview</caption>
+      <image type="source" width="16" height="9">
+        https://github.com/th-ch/youtube-music/assets/16558115/53efdf73-b8fa-4d7b-a235-b96b91ea77fc</image>
+    </screenshot>
+  </screenshots>
+  <releases>
```

Comment:
Do we need to write this XML manually for each release?

Reply:
If you have a sane way of feeding a tool, writing a script that modifies xml is pretty straightforward. Problem usually is the changelog, version number etc.

Reply:
I used this tool to automatically fetch and populate the release history.

https://jakobdev.codeberg.page/work/app/jdAppStreamEdit/

I didn't have to do anything manually so I think it should be easy enough to make a script to automatically update the file.